packages/xo-web/src/xo-app/vm-import/esxi/esxi-import.js

```diff
@@ -0,0 +1,262 @@
+                onChange={linkState}
+                placeholder={formatMessage(messages.password)}
+                required
+                value={password}
+              />
+            </InputCol>
+          </Row>
+          <Row>
+            <LabelCol>{_('sslCertificate')}</LabelCol>
+            <InputCol>
+              <input
+                checked={hasCertificate}
+                name='hasCertificate'
+                onChange={toggleCertificateCheck}
+                type='checkbox'
+                value={hasCertificate}
+              />
+            </InputCol>
+          </Row>
+          <div className='form-group pull-right'>
+            <ActionButton
+              btnStyle='primary'
+              className='mr-1'
+              form='esxi-connect-form'
+              handler={connect}
+              icon='connect'
+              type='submit'
+            >
+              {_('serverConnect')}
+            </ActionButton>
+            <Button onClick={reset}>{_('formReset')}</Button>
+          </div>
+        </form>
+      )}
+      {isConnected && (
+        <form id='esxi-migrate-form'>
+          <Row>
+            <LabelCol>{_('vm')}</LabelCol>
+            <InputCol>
+              <Select disabled={isEmpty(vmsById)} onChange={onChangeVm} options={selectVmOptions} required value={vm} />
+            </InputCol>
+          </Row>
+          <Row>
+            <LabelCol>{_('vmImportToPool')}</LabelCol>
+            <InputCol>
+              <SelectPool onChange={onChangePool} required value={pool} />
+            </InputCol>
+          </Row>
+          <Row>
+            <LabelCol>{_('vmImportToSr')}</LabelCol>
+            <InputCol>
+              <SelectSr
+                disabled={pool === undefined}
+                onChange={onChangeSr}
+                predicate={srPredicate}
+                required
+                value={sr}
+              />
+            </InputCol>
+          </Row>
+          <Row>
+            <LabelCol>{_('network')}</LabelCol>
+            <InputCol>
+              <SelectNetwork
+                disabled={pool === undefined}
+                onChange={onChangeNetwork}
+                predicate={networkPredicate}
+                required
+                value={network}
+              />
+            </InputCol>
+          </Row>
+          {vm !== undefined && (
+            <div>
+              <hr />
+              <h5>{_('vmsToImport', { nVms: 1 })}</h5>
+              <VmData data={vmData} pool={pool} />
+            </div>
+          )}
+          <div className='form-group pull-right'>
+            <ActionButton
+              btnStyle='primary'
+              className='mr-1'
+              disabled={vm === undefined}
+              form='esxi-migrate-form'
+              handler={importVm}
+              icon='import'
+              redirectOnSuccess={
+                vmData === undefined ? undefined : `/home?s=name_label:${encodeURIComponent(vmData.nameLabel)}&t=VM`
```

Comment:
Why don't you redirect to the VM page instead? Also, the name of the VM might not be enough to filter that one VM.